### PR TITLE
Stub for BNFet211 and Update two other mss.

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a stub for a Tārika-Walda-ʾAmid manuscript as discussed in https://github.com/BetaMasaheft/Documentation/issues/2710 and I made updates to other Walda-ʾAmid-manuscripts, that belong to the Beta recension.

### CODE REVIEW

File: ParisBNF/et/BNFet211.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/></support>
+                                <condition key="good"/>
+                            </supportDesc>
+                        </objectDesc>     
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">Very small raquq script with no more than 2 or 3 mm height.</handNote>
```
Comment (thea-m) [inline ParisBNF/et/BNFet211.xml:46]: raqiq?

File: ParisBNF/et/BNFet211.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </encodingDesc>
+        <profileDesc>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Geez</language>
```
Comment (thea-m) [inline ParisBNF/et/BNFet211.xml:73]: As far as I know, it was once decided that this should always be spelled Gǝʿǝz

**Review by thea-m — CHANGES_REQUESTED** [review]
Great, thank you!

**Review by thea-m — APPROVED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
I want to add, that images for BNFet211 are availabe at 
https://gallica.bnf.fr/ark:/12148/btv1b525068170.r=%C3%A9thiopien%20211?rk. 

Is it useful and allowed to include the link to the manuscripts somewhere?

**Comment by thea-m** [comment]
Yes, that would be wonderful! There is a section here on how to add BnF image links: https://betamasaheft.eu/Guidelines/?q=images&id=images

**Comment by CarstenHoffmannMarburg** [comment]
> Yes, that would be wonderful! There is a section here on how to add BnF image links: https://betamasaheft.eu/Guidelines/?q=images&id=images

Thank you for the hint! As far as I understand, the images need to be transformed to a "manifest.json URL" with the ending ".json". I cannot find such an URL on the BNF website, which is hosting the images:

grafik

Is one of these numbers the correct URL or do I need to look up another place?

**Comment by eu-genia** [comment]
Can you check Example 5 in the Guidelines? Thank you.

**Comment by eu-genia** [comment]
(and then compare what you see in the guidelines to the link you posted above)

File: ParisBNF/et/BNFet211.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
                         <collection>Manuscrits orientaux</collection>
                         <collection>Fonds éthiopien</collection>
                         <collection>Collection Mondon-Vidailhet</collection>
-                        <idno>BnF Éthiopien 221</idno>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b525068170.r=%C3%A9thiopien%20211?rk">BnF Éthiopien 221</idno>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet211.xml:29]: the link is https://gallica.bnf.fr/ark:/12148/btv1b525068170

**Pull request merged**